scripts/jest/ts-preprocessor.js

```diff
@@ -3,13 +3,16 @@
 var fs = require('fs');
 var path = require('path');
 var ts = require('typescript');
+var os = require('os');
 
-var tsOptions = {module: 'commonjs'};
+var tsOptions = {
+  module: 'commonjs',
```

Comment:
You need to use `ModuleKind`, not a `string` now: https://github.com/Microsoft/TypeScript/blob/master/src/compiler/types.ts#L2456 + https://github.com/Microsoft/TypeScript/blob/master/src/compiler/types.ts#L2535-L2543